conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -80,6 +81,16 @@ def remove_credentials(self, conjurrc):
         """
         Method for removing user credentials in the system's keyring
         """
-        logging.debug(f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
-        KeystoreAdapter.delete_password(conjurrc.conjur_url)
-        logging.debug(f"Successfully removed credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        logging.debug(
+            f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        for attr in KEYSTORE_ATTRIBUTES:
+            try:
+                KeystoreAdapter.delete_password(conjurrc.conjur_url, attr)
+            # Catches when credentials do not exist in the keyring. If the key does not exist,
+            # the user has already logged out. we still try to remove other leftovers
+            except KeyringDeletionError as err:
```

Comment:
I am not a fan of `Keyring` in the name because today it is keyring and tomorrow it is called `Matanring`. Let's change it to `CredentialStoreDeletionError` and the other errors should follow suit. WDYT?

Reply:
Theses are keyring adapter errors. and not credentialProviders error. so I guess Keyring errors are ok.
I do think we could place these errors in a different file since we do want CLI to be agnostic to Keyring keyword.
WDYT?

Reply:
`KeyringAdapterDeletionError` then? It is just important to know where it originates no?

Reply:
cool. done